### ISSUE TITLE
Show Navigation list when leaderboard open

### ISSUE THREAD

**Issue by randombyte-developer** [taddeimania/lctva#23]
After I click on leaderboard the navigation list disappears so I have to go back with built in browser navigation. This could be a pain when browsed some time with the "previous" and "next" buttons.
This is really low priority for me.

**Comment by taddeimania** [comment]
have you tried the top right navigation drop down to go back to dashboard? might be an issue of not obvious UI on this one.

**Comment by randombyte-developer** [comment]
Yes, that works. But not so obvious. As I said, low priority for me.

**Comment by taddeimania** [comment]
gotcha low priority for me too. Maybe if a user is logged in, show the side nav, if not don't.
